config/database.yml.github

```diff
@@ -0,0 +1,89 @@
+# PostgreSQL. Versions 9.3 and up are supported.
```

Comment:
just curious whu this file ends with `.github`

Reply:
@bronzdoc This file is getting used within the GitHub Actions only, so that is why it has that suffix. It shouldn't be used for anything else.